HW3.postman_collection.json

```diff
@@ -0,0 +1,136 @@
+{
+	"info": {
+		"_postman_id": "c06bffa6-3f8f-4c6a-bc61-d2658828508f",
+		"name": "HW3",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "18396141"
+	},
+	"item": [
+		{
+			"name": "Get All Pokemon",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"var jsonData = JSON.parse(responseBody);\r",
+							"pm.collectionVariables.set(\"pokemonResults\", jsonData.results);"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "https://pokeapi.co/api/v2/pokemon/?limit=30",
+					"protocol": "https",
+					"host": [
+						"pokeapi",
+						"co"
+					],
+					"path": [
+						"api",
+						"v2",
+						"pokemon",
+						""
+					],
+					"query": [
+						{
+							"key": "limit",
+							"value": "30"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Pikachu",
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							"const pokemonList = pm.collectionVariables.get(\"pokemonResults\");\r",
+							"const pikachu = pokemonList.find(pokemon => pokemon.name === \"pikachu\");\r",
+							"pm.collectionVariables.set(\"pikachuUrl\", pikachu.url);"
```

Comment:
It would be better to add some checks (e.g. if - else) to make sure that pikachu isn't undefined.